core/client/src/main/java/alluxio/client/file/FileSystemUtils.java

```diff
@@ -156,7 +156,9 @@ public Boolean apply(Void input) {
         try {
           return fs.getStatus(uri).isPersisted();
         } catch (Exception e) {
-          throw Throwables.propagate(e);
+          Throwables.propagateIfPossible(e);
+          throw new RuntimeException(e);
+          //throw Throwables.propagate(e);
```

Comment:
@jzyhywxz  I think we can use `propagateIfPossible` for now, which should be better than `throw Throwables.propagate(e);`. 
Can you delete the commented out code?